examples/simple.js

```diff
@@ -1,17 +1,224 @@
+fs = require('fs');
```

Comment:
Is this an npm module? It's need to be included in the package.json if it's a dependency

Reply:
fs is node's in-built filesystem. I don't think it needs to be included anywhere. Ref: https://stackoverflow.com/questions/24594796/node-js-npm-install-fs-error

Reply:
makes sense